ajax/libs/colors/package.json

```diff
@@ -1,14 +1,17 @@
 {
   "name": "colors",
-  "filename": "colors.min.css",
-  "version": "1.0",
-  "description": "Better colors for the web. A collection of skin classes for faster prototyping and nicer looking sites.",
+  "filename": "js/colors.min.js",
```

Comment:
This filename change should come with version 3.0.0, and mention that in the commit message. I don't want a single revision failed the test.